docs/core/prerequisites.md

```diff
@@ -0,0 +1,453 @@
+---
+title: Prerequisites for .NET Core on Windows, Linux, and macOS
+description: Supported Windows, Linux, and macOS versions and .NET Core dependencies to develop, deploy, and run .NET Core applications.
+author: leecow
+ms.author: leecow
+ms.date: 10/30/2019
+zone_pivot_groups: operating-systems-set-one
+---
+
+# Prerequisites for .NET Core
+
+This article provides information on getting started with .NET Core. .NET Core can be installed on Windows, Linux, and macOS. You can code in your favorite text editor and produce cross-platform libraries and applications.
```

Comment:
"This article provides information on getting started with .NET Core. " -> this seems misleading. This is not what this topic should be about.

Reply:
Ahh yes good catch. I think that is what it *will* become. I'll adjust it.

Reply:
It shouldn’t become that. That’s not what a getting started with Core looks like.